SumoLogic.Logging.NLog/BufferedSumoLogicTarget.cs

```diff
@@ -91,36 +89,40 @@ public BufferedSumoLogicTarget(ILog log, HttpMessageHandler httpMessageHandler)
         [SuppressMessage("Microsoft.Design", "CA1056:UriPropertiesShouldNotBeStrings", Justification = "Property needs to be exposed as string for allowing configuration")]
         public string Url
         {
-            get;
-            set;
+            get { return (_urlLayout as SimpleLayout).Text; }

```

Comment:
I don't understand why url need support layout. this should be a static value

Reply:
Many times you want your custom settings in one location independent of environment. This could mean that the URL is stored in appsettings.json / app.config or maybe retrieved from some magic secret coffin in the cloud. Having it hardcoded in the NLog.config removes such options.